### ISSUE TITLE
feature request: option to install plugin with default course/config settings applied

### ISSUE THREAD

**Issue by moloko** [adaptlearning/adapt-cli#94]
appreciate this is a bit of an ask - but it would be great if when I install I plugin I could have it auto-populate config.json/course.json with the default settings (as defined in properties.schema)...!

**Comment by chris-steele** [comment]
Yeah nice idea, I'll give it some thought and get back to you (hopefully before another 8 months passes!)
